apps/coscrad-frontend/src/components/resources/shared/resource-preview.tsx

```diff
@@ -0,0 +1,16 @@
+import { AggregateType } from '@coscrad/api-interfaces';
+
+interface ResourcePreviewProps {
+    resourceType: AggregateType;
+}
+
+const mediaUrl = 'https://kaaltsidakah.net/coscrad/resource-icons';
```

Comment:
We need
- a set of default icons (Justin help?)
    - available on a public media server (COSCRAD public facing website server CMS?)
- config property to override any or all of these for each group
     - validation and Jest test
     - Cypress test that the config is being used